core/core_integrations/telegram.js

```diff
@@ -1,15 +1,7 @@
-var options = {
-    // Used by the monkey patch to node-telegram-bot-api
-    stopPolling: true
-},
-    TelegramBot = require.safe('node-telegram-bot-api'),
-    bot = null,
+var bot = null,
+    api = null,
     shim = require('../shim.js'),
```

Comment:
`require.once` - I know it is old code but this might prevent telegram based bots from being able to hotswap code on a restart